src/themes/greyVest/index.js

```diff
@@ -861,3 +942,27 @@ export let QueryBuilder = defaultProps({ Button, MissingTypeComponent })(
 export let SearchFilters = defaultProps({ QueryBuilder, FiltersBox })(
   BaseSearchFilters
 )
+
+export let FilterButtonList = defaultProps({
+  Button,
+  CheckButton,
+  Icon,
+  Modal,
+  className: 'gv-filter-button-list',
+})(BaseFilterButtonList)
+
+export let StepsAccordion = defaultProps({
+  Button,
+  Icon,
+  className: 'gv-query-wizard',
```

Comment:
is this right? I see you are also adding it to QueryWizard

Reply:
ah, this was a holdover from splitting out StepsAccordion. I changed it to `gv-steps-accordion` and removed the QueryWizard one, since QueryWizard just returns a StepsAccordion anyway.

I suppose we could add a wrapper div to QueryWizard so we can give it separate class/style props, but I don't think there's any real use case for that now (and it would be trivial to add if it ever becomes necessary)